tests/benchmarks.py

```diff
@@ -175,7 +175,7 @@ def bench_download_50_1gb(adl, lpath, rpath, config):
             'bench_upload_50_1gb': {
                 'nthreads': 64,
                 'buffersize': 32 * 2**20,
```

Comment:
I assume this is still ideal. My updates focused only on downloads. I haven't had an opportunity to re-run my script that looks for the optimal parameters.

In the future, there still may be an internal buffer for uploads. The buffered uploader has more edge cases than the downloader. Ultimately, the uploader implementation could be simplified and faster, but it may take more exploration.